<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenconnectProp</class>
 <widget class="QWidget" name="OpenconnectProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenConnect Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="grp_general">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Gateway:</string>
        </property>
        <property name="buddy">
         <cstring>leGateway</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leGateway"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;CA Certificate:</string>
        </property>
        <property name="buddy">
         <cstring>leCaCertificate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="leCaCertificate"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Proxy:</string>
        </property>
        <property name="buddy">
         <cstring>leProxy</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="leProxy"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>CSD &amp;Wrapper Script:</string>
        </property>
        <property name="buddy">
         <cstring>leCsdWrapperScript</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="chkAllowTrojan">
        <property name="text">
         <string>Allow Cisco Secure Desktop &amp;trojan</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KUrlRequester" name="leCsdWrapperScript"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Certificate Authentication</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;User Certificate:</string>
        </property>
        <property name="buddy">
         <cstring>leUserCert</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Private &amp;Key:</string>
        </property>
        <property name="buddy">
         <cstring>leUserPrivateKey</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="leUserCert"/>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="leUserPrivateKey"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="chkUseFsid">
        <property name="text">
         <string>Use &amp;FSID for key passphrase</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
